examples/express-helmet/src/app.ts

```diff
@@ -0,0 +1,71 @@
+import { createYoga, createSchema } from 'graphql-yoga'
+import express from 'express'
+import helmet from 'helmet'
+
+export function buildApp(app: ReturnType<typeof express>) {
+  const graphQLServer = createYoga({
+    schema: createSchema({
+      typeDefs: /* GraphQL */ `
+        scalar File
+        type Query {
+          hello: String
+        }
+        type Mutation {
```

Comment:
I think we can remove these extras like file upload etc, right?

Reply:
Sure, no need to overcomplicate it, you're right